docs/pages/connect-your-client/web-ui.mdx

```diff
@@ -31,3 +31,22 @@ You must have the `join_sessions` allow policy in a role you've been assigned to
 For information about how to configure the `join_sessions` allow policy and participant modes for a role, see 
 [Configure an allow policy](../access-controls/guides/moderated-sessions.mdx#configure-an-allow-policy).
 
+## Secure Copy (SCP) files with the Web UI
+
+Teleport provides two methods to copy local files to a remote system through the web interface.
+
+### Teleport UI
+
+You can use the upload functionality:
+
+![Teleport UI](../../img/download-files.png)
+
+You can also upload files from a system using drag and drop:
+
+![drag and drop](../../img/upload-ui.png)
+
+### Teleport Connect
```

Comment:
Thoughts on moving changes from this file to the introduction page you link to in the PR description? https://goteleport.com/docs/connect-your-client/introduction/#protocols

web-ui.mdx is largely for the Web UI, so it feels a bit awkward to mention Connect there.

I guess another problem is that we already have dedicated pages for Connect and the Web UI, so the question is whether these instructions should go to those specific pages individually or if they should be grouped together.

For the start though, I don't think there would be anything wrong with keeping them grouped, like we already do on the introduction page.

Reply:
Thanks! This may work better on the protocol page. Perhaps a new tabbed section for the **Transferring files (scp)** paragraph. The WebUI page is rather sparse in its current state, but that could be addressed separately.